gap/Paths/paths.gi

```diff
@@ -1302,3 +1302,46 @@ InstallMethod( ViewObj, "for an edge-coloured edge-face-path",
         fi;
     end
 );
+
+
+#######################################
+##
+##      Waists
+##
+
+InstallMethod( AllTwoWaistsOfComplex, "for a simplicial surface",
+    [IsTwistedPolygonalComplex],
```

Comment:
In the line before you wrote that the method is just for simplicial surfaces that is not uniformly.